app/Services/Manager.php

```diff
@@ -254,6 +254,32 @@ public function appendReportback($data, $parameters)
     }
 
     /**
+     * Get the top three reportbacks from a given leaderboard.
+     *
+     * @param array $leaderboard
+     * @return array $topThree
+     */
+    public function getTopThreeReportbacks($leaderboard)
+    {
+        $topThreeUsers = array_slice($leaderboard, 0, 3);
+        $places = ['1st', '2nd', '3rd'];
+
+        foreach ($topThreeUsers as $key => $user) {
+            $latestReportback = array_pop($user->reportback->reportback_items->data);
```

Comment:
Super nit-picky, but could we call this variable `$latestReportbackItem` to really drive the point home it's the item and not the general reportback? Try to make sure the terminology is applied across projects as much as possible cause there's already enough confusion between Reportbacks and Reportback Items @_@